chains/evm/listener/listener.go

```diff
@@ -17,10 +17,23 @@ import (
 var BlockRetryInterval = time.Second * 5
 var BlockDelay = big.NewInt(10) //TODO: move to config
 
+// DepositLogs struct holds event data with all necessary parameters and a handler response
+// https://github.com/ChainSafe/chainbridge-solidity/blob/develop/contracts/Bridge.sol#L47
 type DepositLogs struct {
+	// ID of chain deposit will be bridged to
 	DestinationID uint8
-	ResourceID    [32]byte
-	DepositNonce  uint64
+	// ResourceID used to find address of handler to be used for deposit
+	ResourceID [32]byte
+	// Nonce of deposit
+	DepositNonce uint64
+	// Address (msg.sender: user)
+	Address common.Address
```

Comment:
i think it will be more meaning-full to rename it to SenderAddress or something like that